pymc_bart/bart.py

```diff
@@ -88,6 +91,7 @@ def __new__(
         Y: TensorLike,
         m: int = 50,
         alpha: float = 0.25,
+        response: str = "mix",
```

Comment:
Currently, there is no logic to handle this. So we only have linear or constant right? I think this is fine, let's keep only those two options, at least for this PR.

Reply:
added in https://github.com/pymc-devs/pymc-bart/pull/79/commits/996596f2053bd4bb1b942a5b483d712d84b042c0 as in the original PR